### ISSUE TITLE
Unstable loss during Kontext lora training

### ISSUE THREAD

**Issue by Twinkle-ce** [modelscope/DiffSynth-Studio#964]
During Kontext LoRA training (learning rate = 1e-5, rank = 64), the training loss shows large fluctuations across epochs, sometimes dropping to ~0.02 and then surging above 0.7 within the same epoch. Is this degree of instability normal for LoRA training？

<img width="2332" height="283" alt="Image" src="https://github.com/user-attachments/assets/9e7f0705-71ab-4f43-97df-5e0bffb02132" />

**Comment by Artiprocher** [comment]
@Twinkle-ce The loss value depends on the timestep; during each training step, a timestep is randomly sampled and the loss is computed accordingly. Oscillations in the loss are therefore normal. Please use a moving average to obtain a smoothed loss curve.
